### ISSUE TITLE
compute - cspell found spelling errors in public API surface

### ISSUE THREAD

**Issue by danieljurek** [Azure/azure-sdk-for-net#23174]
Investigation for this PR: https://github.com/Azure/azure-sdk-for-net/pull/23150 found spelling errors. When the linked PR merges this service will be opted out of public API surface spell checking to keep other PRs unblocked. This issue describes how to opt back in. 

If the above linked PR has not merged this service is still opted out and not blocked by the detection of spelling errors in the public API surface. 

## What to do

1. Correct spelling errors or check in fixes for the false positives 
1. Opt back into Public API surafe spell checking

## Correct errors

To correct false positives or learn about spell checking see: https://aka.ms/azsdk/engsys/spellcheck

#### Spell Check Output

```
1/1 .\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs 1380.18ms X
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:41:62 - Unknown word (Skus)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:41:85 - Unknown word (Skus)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:688:34 - Unknown word (Skus)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:690:27 - Unknown word (Skus)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:869:161 - Unknown word (skus)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:870:195 - Unknown word (skus)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:871:212 - Unknown word (skus)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:872:246 - Unknown word (skus)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:917:169 - Unknown word (vmss)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:918:203 - Unknown word (vmss)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:921:186 - Unknown word (vmss)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:922:220 - Unknown word (vmss)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:1077:77 - Unknown word (Reimage)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:1077:99 - Unknown word (Reimage)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:1077:215 - Unknown word (Reimage)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:1077:243 - Unknown word (Reimage)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:1720:36 - Unknown word (Unattend)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:1722:26 - Unknown word (Unattend)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:1923:9 - Unknown word (Dcos)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:1925:9 - Unknown word (Kubernetes)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:2327:92 - Unknown word (SSDLRS)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:2328:89 - Unknown word (SSDLRS)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:3082:67 - Unknown word (VCPUs)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:3486:88 - Unknown word (SSDLRS)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:3487:85 - Unknown word (SSDLRS)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:3672:113 - Unknown word (Vhds)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:3674:30 - Unknown word (Vhds)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:4459:96 - Unknown word (Unattend)
D:\code\azure-sdk-for-net\sdk\compute\Azure.ResourceManager.Compute\api\Azure.ResourceManager.Compute.netstandard2.0.cs:4459:123 - Unknown word (Unattend)
CSpell: Files checked: 1, Issues found: 29 in 1 files
```

To check the public API surface locally, in the root of the repo run:

```
eng/scripts/Export-API.ps1 -ServiceDirectory compute -SpellCheckPublicApiSurface
```

You may also need to set `-SDKType` if the appropriate value of `SDKType` is not `all` for the service.

## Opt back in

In the `ci.yml` file for this service, remove the line:

```yaml
SpellCheckPublicApiSurface: false
```

**Comment by yungezz** [comment]
same issue with #23188

**Comment by ArcturusZhang** [comment]
@danieljurek sorry for the delay of response.
Previously we are focusing to get the new version of track 2 compute SDK merged into main. And now it is merged.
I run the command
```
eng/scripts/Export-API.ps1 -ServiceDirectory compute -SpellCheckPublicApiSurface
```
on my local repo after the new track 2 is merged. It does not give me any error. Therefore I suppose I just need to remove the ignore clause of compute RP in the `cspell.json` file. I will open a PR soon about this.

**Comment by ArcturusZhang** [comment]
I opened [this PR](https://github.com/Azure/azure-sdk-for-net/pull/23637) to remove compute from the ignore list

**Comment by ArcturusZhang** [comment]
Also I opened [this PR](https://github.com/Azure/azure-sdk-for-net/pull/23638) to fix the spellcheck in new track 2 SDK of keyvault.
This issue does not exist when you check this, therefore it does not have a corresponding issue now
